ledger/sandbox-common/src/main/scala/platform/sandbox/cli/CommonCliBase.scala

```diff
@@ -195,6 +195,15 @@ class CommonCliBase(name: LedgerName) {
           )
         )
 
+      opt[TlsVersion]("tls-version")
```

Comment:
I suggest calling that flag "min-tls-version" to drive the point home that it is not just a single version that is supported.

Reply:
Ok